src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -32,24 +32,24 @@
 abstract class DoctrineType extends AbstractType
 {
     /**
-     * @var ManagerRegistry
+     * @var DoctrineChoiceLoader[]
      */
-    protected $registry;
+    private static $choiceLoaders = array();
```

Comment:
well, see my previous comments (on the hidden discussion): making it static brings no benefit (as several instances will not share the same cache keys), and will leak memory by keeping the cached loaders in memory until the end of the process rather than until the instance is destroyed (which can make a huge difference in tests, where each test gets a new container, and so a different set of cached data)